radixdlt/babylon-wallet-android
# [ABW-3645] - Seed phrase UI refinements

[Jira Ticket](https://radixdlt.atlassian.net/browse/ABW-3645)

## Description
This PR refines the UI mainly for the Seed Phrase entry screen:

- General spacing and UI style adjustments
- Removed Advanced Mode button for Babylon seed phrase
- Removed seed phrase words suggestions and autocompletion
- Fixed brief content flickering when entering `RestoreMnemonicsScreen`
- Ensured the focused word in the seed phrase screen is above the keyboard when typing
- Hidden keyboard when fully scrolled to the bottom in the seed phrase screen
- Added loading state to the skip button in `RestoreMnemonicsScreen`
- Updated transition animations related to the seed phrase screen
- Fixed content transition animations within `RestoreMnemonicsScreen`


## How to test

Mnemonic restoring from a backup:
1. Fresh install or factory reset the wallet
2. Choose Restore Wallet
3. Connect to Google Drive and select a backup (preferably with multiple seed phrases)
4. Test the general look & feel, especially for the seed phrase input screen

Mnemonic restoring for accounts recovery:
1. Fresh install or factory reset the wallet
2. Choose Restore Wallet
3. Skip Google Drive -> Other Restore Options -> Recover Control with Main Seed Phrase
4. Test the general look & feel, especially for the seed phrase input screen

Mnemonic restoring for Olympia wallet import:
1. Ensure the selected gateway is Mainnet
2. Go to Settings -> Troubleshooting -> Import from a Legacy Wallet
3. Scan an Olympia wallet accounts export QR
4. Test the general look & feel, especially for the seed phrase input screen

## PR submission checklist
- [ ] I have tested all of the above
